test/scripts/e2e_subs/create_destroy_optin_optout.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+# This script creates some state required for Indexer testing
+# TIMEOUT=400
+
+date '+create_destroy_optin_optout start %Y%m%d_%H%M%S'
+
+set -e
+set -x
+set -o pipefail
+export SHELLOPTS
+
+WALLET=$1
+
+# Directory of this bash program
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+gcmd="goal -w ${WALLET}"
+
+ACCOUNT=$(${gcmd} account list | awk '{ print $3 }')
+ACCOUNTB=$(${gcmd} account new | awk '{ print $6 }')
+${gcmd} clerk send -a 1000000 -f $ACCOUNT -t $ACCOUNTB
+
+#############################################
+# Account - create / close / create / close #
+#############################################
+ACCOUNT_NEW=$(${gcmd} account new | grep "Created" | awk '{ sub("\r", "", $NF); print $NF }')
+${gcmd} clerk send -a 100000 -f $ACCOUNT -t $ACCOUNT_NEW
+${gcmd} clerk send -a 0 -f $ACCOUNT_NEW -t $ACCOUNT -c $ACCOUNT
+${gcmd} clerk send -a 100000 -f $ACCOUNT -t $ACCOUNT_NEW
+${gcmd} clerk send -a 0 -f $ACCOUNT_NEW -t $ACCOUNT -c $ACCOUNT
+
+#####################################
+# Account - create / close / create #
+#####################################
+ACCOUNT_NEW=$(${gcmd} account new | grep "Created" | awk '{ sub("\r", "", $NF); print $NF }')
```

Comment:
And I think this would be clearer with the same `account new` incantation as ACCOUNTB, but probably titled here ACCOUNTD just to keep clear that it's a different account than the previous ACCOUNT_NEW